resources/building-mobile-extensions/get-the-application-context.md

```diff
@@ -0,0 +1,36 @@
+# Get the Application Context
+
+In Android, you can retrieve the `android.app.Application` instance using the `MobileCore.getApplication()` API. The returned Application represents the base class which maintains the global application state. For more information, [Moble Core API Reference](../../using-mobile-extensions/mobile-core/configuration-reference/mobile-core-api-reference.md)
+
+{% tabs %}
+{% tab title="Android" %}
+
+### Android
+
+### getApplication
+
+{% hint style="warning" %}
+
+`MobileCore.getApplication` may return `null` if the Application object was destroyed or if `MobileCore.setApplication`was not previously called.
```

Comment:
- Please change "may" to "might".
- Does "Application" have to be capitalized? If not, please change to a lower case "a".